client/src/components/NewsOfTheDays/style.scss

```diff
@@ -0,0 +1,136 @@
+@import '../styles/variables';
+@import '../styles/fonts';
```

Comment:
we discussed it today's morning. If u forgot -- ask Vitalii

Reply:
removed